style.css

```diff
@@ -115,4 +115,29 @@
 .price{
   margin-top:15px;           /*to have a space between (price) and (p)*/
   color: #2aa6af;            
-}
\ No newline at end of file
+}
+.qout-container{
+  height: 400px;
+  width: auto;
+  color: white;
```

Comment:
The colour of the text is white and the background colour also close to white, which made the text difficult to read 
Pay attention to colour contrast